src/Facade/ResourceBuilders/StockLocatorResourceBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Linn.Stores.Facade.ResourceBuilders
+{
+    using System;
+
+    using Linn.Common.Facade;
+    using Linn.Stores.Domain.LinnApps.StockLocators;
+    using Linn.Stores.Resources;
+
+    public class StockLocatorResourceBuilder : IResourceBuilder<StockLocator>
+    {
+        public StockLocatorResource Build(StockLocator stockLocator)
+        {
+            if (stockLocator == null)
```

Comment:
Does this ever get called with a null stock locator?

Reply:
Not anymore. I was returning a null response in the delete case initially, but I changed it to return the deleted item. Good spot